frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -889,7 +889,7 @@ function FileManagerMenu:getStartWithMenuTable()
     end
     return {
         text_func = function()
-            local start_with = G_reader_settings:readSetting("start_with")
+            local start_with = G_reader_settings:readSetting("start_with") or "filemanager"
```

Comment:
I was going to say `G_reader_settings:readSetting("start_with", "filemanager")` is also an option now, but since we don't actually do anything write-related to the settings here, this is probably still clearer.